### PR TITLE
feat: activities initial

### PR DESCRIPTION
**Pull request by donaldosebas**

### CODE REVIEW

**Review by raulangelj — DISMISSED** [review]
LGTM! 🚀

File: src/navigations/AppNavigator/index.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const RootStack = createNativeStackNavigator<RootStackParamList>()
 
 const AppNavigator: React.FC = () => {
   const screenForAppState = useCallback(() => {
-    return [SignIn(), MainBottonNavigator()]
+    return [MainBottonNavigator(), Activities()]
```
Comment (raulangelj) [inline src/navigations/AppNavigator/index.tsx:16]: It is missing the Sign In group! Please fix it

**Review by raulangelj — CHANGES_REQUESTED** [review]

File: src/navigations/AppNavigator/index.tsx (hunk 1 above)
Comment (donaldosebas) [reply src/navigations/AppNavigator/index.tsx:16]: shut up and merge it

Comment (raulangelj) [reply src/navigations/AppNavigator/index.tsx:16]: No fix, no merge. Please, fix it.

Comment (donaldosebas) [reply src/navigations/AppNavigator/index.tsx:16]: no

Comment (donaldosebas) [reply src/navigations/AppNavigator/index.tsx:16]: merge it

**Review by raulangelj — APPROVED** [review]
LGTM! 🚀

**Pull request merged**